storage/remote/queue_manager.go

```diff
@@ -57,6 +57,7 @@ type queueManagerMetrics struct {
 	enqueueRetriesTotal   prometheus.Counter
 	sentBatchDuration     prometheus.Histogram
 	highestSentTimestamp  *maxGauge
+	highestRecvTimestamp  *maxGauge
```

Comment:
this metric should not be a part of the queue manager metrics, we should just pass a reference to the metric from write storage when we create the queue manager

Reply:
Done. Hence, resolving.